mainapp/views.py

```diff
@@ -44,6 +51,144 @@ class CreateRequest(CreateView):
+                        "4494e2e2-a125-44c7-8231-117f5fb7fe4a": [
+                             self.object.detailfood
+                        ],
+                        "bdd7d711-9c95-4d2c-a8bb-ffa796b306cc": [
+                             self.object.detailcloth
+                        ],
+                        "ac8bb683-d0e8-4c51-a7b5-6628b7247fbd": [
+                             self.object.detailmed
+                        ],
+                        "42cc5aa3-d946-46b8-a119-84417bbc2942": [
+                             self.object.detailkit_util
+                        ],
+                        "dffd9506-3b55-4bfc-8659-34793c3ce938": [
+                             self.object.detailtoilet
+                        ],
+                        "a73cdc00-df3a-4317-a8b6-583907f55d0f": [
+                             self.object.requestee
+                        ],
+                        "e12f60d9-2503-4064-865f-7ce2fcafabe3": [
+                            "New"
+                        ],
+                        "2820b045-dad5-44d0-b917-6e7b16917c99": [
+                            pretty(self.object.is_request_for_others)
+                        ],
+                        "486a202c-48d6-4b30-aeb1-098a079210cf": [
+                            pretty(self.object.needrescue)
+                        ],
+                        "2456a552-dc48-422d-a8ba-16fe8c6c560b": [
+                            pretty(self.object.needwater)
+                        ],
+                        "bd129f3e-0749-4946-aa27-2ee2224376aa": [
+                            pretty(self.object.needfood)
+                        ],
+                        "b0f6c545-6d21-4702-aee5-f9c1def09d15": [
+                            pretty(self.object.needcloth)
+                        ],
+                        "c8833ed3-8a82-4aea-bf26-af7a164cf119": [
+                            pretty(self.object.needmed)
+                        ],
+                        "40ceb20a-0f4b-4d23-803e-bdd527d9768b": [
+                            pretty(self.object.needkit_util)
+                        ],
+                        "b96e5ada-f754-46f6-88a1-55706ea73fea": [
+                            pretty(self.object.needtoilet)
+                        ],
+                        "667e5f75-70b6-455a-8f5c-73779620c282": [
+                            self.object.latlng_accuracy
+                        ],
+                        "ea8a2458-17d3-47f9-ae44-20e910af941e": [
+                        {
+                        "lon": lon,
+                        "lat": lat
+                        }
+                        ],
+                    },
+
+        'type': 'report',
+        'author_email' : "data@keralarescue.in",
+        'updated': str(self.object.dateadded) ,
+        'url': 'http://35.200.150.137/platform/api/v3/posts/{}'.format(self.object.id),
+
+        }
+        
+
+
+        out = requests.post(post_url, json=post_data, headers=headers)
```

Comment:
Can we wrap this in a `try` / `catch` - log any errors, but still return successfully to the user so that the data still gets saved in our db in case the request to Ushahidi fails?

Reply:
The data is saved initially

Reply:
I'll add the try catch thou